app/models/post.rb

```diff
@@ -5,13 +5,18 @@ class Post < ApplicationRecord
 
   after_save :update_posts_counter
 
+  validates :title, length: { maximum: 250,
+                              too_long: '250 characters is the maximum allowed' }
+
+  validates(:comments_counter, :likes_counter, numericality: { only_integer => true, greater_than_or_equal_to: 0 })
+
```

Comment:
- [ ] Great job so far adding the validation 👏. However, when I first ran the tests, I got this error: `undefined local variable or method only_integer`.  So the fix I found is that after the `only_integer` key, use a `:` instead of a `=>`. Please update that to ensure that we can run your tests. 


```suggestion

  validates(:comments_counter, :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 })

```

Reply:
@ridwanediallo  yes. I made that change because I was getting linter error in github because of the column and I did not check again running the tests. I put back the column (:) and surprise, surprise, I am not having any errors!

Reply:
properly resolved! ✔️